ee_tests/src/page_objects/space_pipeline_tab.page.ts

```diff
@@ -162,13 +80,14 @@ export class PipelineDetails extends BaseElement {
     return Promise.resolve(pipelines);
   }
 
-  public async approvalRequired(): Promise<boolean> {
-    return this.isElementPresent(by.cssContainingText('a', 'Input Required'));
+  public async isInputRequired(): Promise<boolean> {
+    return this.inputRequired.isDisplayed();
```

Comment:
This line is causing
```
Failed: No element found using locator: by.cssContainingText("a", "View Log")
```
It tries to access Jenkins log immediately, I don't understand why. If you replace it with `return this.inputRequired.isPresent();` it starts waiting for pipeline to finish correctly.